pom.xml

```diff
@@ -1067,6 +1054,18 @@ under the License.
 						</plugin>
 					</plugins>
 				</pluginManagement>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<excludedGroups>
+								${github-actions.surefire.excludedGroups},
```

Comment:
Interesting that this doesn't fail if the property is empty (i.e., theres a leading comma).